catalog/core/catalog-core-commons/src/main/java/ddf/catalog/filter/delegate/ValidationQueryDelegate.java

```diff
@@ -57,17 +55,20 @@ public Boolean not(Boolean operand) {
 
     @Override
     public Boolean propertyIsEqualTo(String propertyName, String literal, boolean isCaseSensitive) {
-        return propertyName.equals(VALIDATION_ERRORS) || propertyName.equals(VALIDATION_WARNINGS);
+        return propertyName.equals(BasicTypes.VALIDATION_ERRORS)
```

Comment:
The comment refers to the `and`, `or` and `not` methods. Not sure if that implies any kind of behavior for the `propertyIs...` method or not. I kind of see that `propertyIsEqualTo()` can be hard-coded to check whether the property is `VALIDATION_ERROR` or `VALIDATION_WARNING` (although I wonder what callers would pass as `literal` and why it's being ignored). But I can't wrap my head around the meaning of the two other methods...